src/main/java/me/pwns/logistica/world/regen/BlockGroupContainer.java

```diff
@@ -0,0 +1,22 @@
+package me.pwns.logistica.world.regen;
+
+public class BlockGroupContainer {
+    private BlockGroup childGroup;
+
+    public BlockGroupContainer(SavedBlockState block) {
+        this.childGroup = new BlockGroup(block, this);
```

Comment:
Kinda weird that we're constructing this from block. Probably better to let the user define their blockgroup, and pass the group itself.